apps/meteor/app/lib/server/methods/sendMessage.ts

```diff
@@ -132,7 +132,7 @@ Meteor.methods<ServerMethods>({
 			ts: Match.Maybe(Date),
 			t: Match.Maybe(String),
 			otrAck: Match.Maybe(String),
-			bot: Match.Maybe(Boolean),
+			bot: Match.Maybe(Object),
```

Comment:
![image](https://github.com/user-attachments/assets/9d6961e0-2f99-46df-a623-962c853ca01c)


(Can we do Match.maybe(boolean | Object)?